radixdlt-java/src/main/java/com/radixdlt/client/core/pow/ProofOfWork.java

```diff
@@ -33,7 +34,7 @@ public void validate() throws ProofOfWorkException {
 		byteBuffer.putInt(magic);
 		byteBuffer.put(seed);
 		byteBuffer.putLong(nonce);
-		String hashHex = Bytes.toHexString(RadixHash.of(byteBuffer.array()).toByteArray());
+		String hashHex = Bytes.toHexString(new Hash(byteBuffer.array()).toByteArray());
```

Comment:
I would prefer `Hash.of` here, but your call

Reply:
agreed! thanks!